crates/wasi-nn/examples/classification-example/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 publish = false
 
 [dependencies]
-wasi-nn = "0.1.0"
+wit-bindgen-rust = { git = "https://github.com/bytecodealliance/wit-bindgen", rev = "a79a4be33d76ddf62839ba71602c26a96610ef7c" }
```

Comment:
Maybe a new version of the `wasi-nn` crate could be created which does `wit_bindgen_rust::import!("spec/wasi-nn.wit.md");` internally? That way uses of the wasi-nn interface don't have to copy/submodule the wasi-nn specification.

Reply:
cc: @brianjjones; I opened https://github.com/bytecodealliance/wasi-nn/issues/64 about this a bit ago. I would hope that `import!` would be a viable approach.

Reply:
> Maybe a new version of the wasi-nn crate could be created which does wit_bindgen_rust::import!("spec/wasi-nn.wit.md"); internally?

maybe.

Reply:
i guess it isn't a big problem as the submodule can be shared with https://github.com/bytecodealliance/wasmtime/tree/main/crates/wasi-nn once it's adapted.